wdio.conf.js

```diff
@@ -48,6 +48,7 @@ exports.config = {
             './e2e/wdio/platform/**/input-group.e2e-spec.ts',
             './e2e/wdio/platform/**/link.e2e-spec.ts',
             './e2e/wdio/platform/**/list.e2e-spec.ts',
+            './e2e/wdio/platform/**/time-picker.e2e-spec.ts'
```

Comment:
please move to platformB. Trying to keep them alphabetical and platformA currently has a longer run time.

Reply:
?